packages/image_picker/image_picker_ios/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -29,10 +29,7 @@ - (instancetype)initWithResult:(nonnull FlutterResultAdapter)result {
 
 #pragma mark -
 
-@interface FLTImagePickerPlugin () <UINavigationControllerDelegate,
```

Comment:
Also moved to `FLTImagePickerPlugin_Test.h` so the tests can directly call the `PHPickerViewControllerDelegate` methods.